### PR TITLE
bgpv2-docs: updating troubleshooting and operations guide

### PR DESCRIPTION
**Pull request by harsimran-pabla**
Adding BGPv2 specific details to BGP operations and troubleshooting guide.

### CODE REVIEW

File: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst [hunk 1]
```diff
@@ -11,6 +11,171 @@ BGP Control Plane Operation Guide
...
+
+You can validate the BGP attributes are advertised based on configured :ref:`CiliumBGPAdvertisement <bgp-adverts>` resources.
+
+
+Policies
+~~~~~~~~
+
+Cilium BGP installs GoBGP policies for managing per peer advertisement and BGP attributes. As this
+is an internal implementation detail, it is not exposed via Cilium CLI. However, for debugging purpose
+can inspect installed BGP policies using cilium-dbg CLI from the Cilium agent pod.
```
Comment (rastislavs) [inline Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst]:
```suggestion
you can inspect installed BGP policies using cilium-dbg CLI from the Cilium agent pod.
```

File: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst [hunk 2]
```diff
@@ -11,6 +11,171 @@ BGP Control Plane Operation Guide
...
+    /home/cilium# cilium-dbg bgp route-policies
+    VRouter   Policy Name          Type     Match Peers      Match Prefixes (Min..Max Len)   RIB Action   Path Actions
+    65001     65000-ipv4-PodCIDR   export   fd00:10::1/128   10.1.0.0/24 (24..24)            accept       AddCommunities: [65000:99]
+    65001     65000-ipv6-PodCIDR   export   fd00:10::1/128   fd00:10:1::/64 (64..64)         accept       AddCommunities: [65000:99]
+    65001     allow-local          import                                                    accept
+
+CiliumBGPNodeConfig Status
+==========================
+
+Cilium node on which BGP control plane is enabled based on `CiliumBGPClusterConfig` node selector gets associated
```
Comment (rastislavs) [inline Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst]:
```suggestion
Each Cilium node on which BGP control plane is enabled based on `CiliumBGPClusterConfig` node selector gets associated
```

**Review by rastislavs — APPROVED** [review]
LGTM, just a few nits

**Review by YutaroHayakawa — APPROVED** [review]
The new contents looks good to me.

**Review by ghost — APPROVED** [review]
lgtm

**Pull request merged**